src/styles/contact.scss

```diff
@@ -0,0 +1,85 @@
+.contacts {
+  padding-inline: 20px;
+  padding-block: 64px;
+
+  @include on-tablet {
+    padding-inline: 72px;
+    padding-block: 120px;
+  }
+
+  @include on-desktop {
+    padding: 120px;
+  }
+
+  &__content {
+    @include page-grid;
+
+  }
+
+  &__form {
+    grid-column: 1 / -1;
+
+    @include on-desktop {
+      grid-column: span 6;
+    }
+  }
+
+  &__input {
+    box-sizing: border-box;
+    width: 100%;
+    padding: 14px 16px;
+    margin-bottom: 16px;
+
+    border-radius: 8px;
+    border: none;
+
+    background-color: #f9f9f9;
+    font-family: inherit;
+    font-size: 14px;
+    line-height: 140%;
+    letter-spacing: 0.15px;
+
+    &::placeholder {
+      color: #7e7e83;
+    }
+
+    &--message {
+      resize: none;
+    }
+  }
+
+  &__address {
+    grid-column: 1 / -1;
+
+    @include on-desktop {
+      grid-column: 8 / 13;
+      padding: 0;
+    }
+  }
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    padding-top: 50px;
+    list-style-type: none;
+
+    @include on-desktop {
+      padding: 0;
+    }
+  }
+
+  &__list-item {
+    margin-bottom: 24px;
+  }
+
+  &__label {
+    color: #9393a3;
+  }
+
+  &__link {
+    display: block;
+    margin-top: 8px;
+    text-decoration: none;
+    color: $color-primary;
```

Comment:
Contact links text colors should change on hover [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Change%20text%20color%20on%20hover%20for%20phone%2C%20email%2C%20and%20address)